verification.py

```diff
@@ -3916,6 +3921,499 @@ def add_random_instance(
         self.add_ms_instance(key, cmd)
 
 
+class MutationTest(SimulationVerifier):
+    def _transition_matrix_chi_sq(self, transitions, transition_matrix):
+        tm_chisq = []
+        for row, p in zip(transitions, transition_matrix):
+            not_zeros = row > 0
```

Comment:
This is a mistake from my side. Should be `not_zeros = p > 0`. Given the mutation rates we are using, this does not change much.